src/components/CustomerQuota/ItemsSelection/Item.tsx

```diff
@@ -1,13 +1,14 @@
-import { PolicyIdentifier } from "../../../types";
-import { ItemNoQuota } from "./ItemNoQuota";
-import { ItemCheckbox } from "./ItemCheckbox";
-import { ItemStepper } from "./ItemStepper";
 import { StyleSheet, View } from "react-native";
 import React, { FunctionComponent, useContext } from "react";
+import { ItemCheckbox } from "./ItemCheckbox";
```

Comment:
Just curious, is this sorting because of prettier or eslint?

Reply:
i did myself lehh, i assume there won't be many `mobile-application` pr to cause merge conflict before it got merged. should I revert it?

Reply:
nope, just wondering!